src/UserService.Business/Commands/User/CreateUserCommand.cs

```diff
@@ -308,7 +308,8 @@ public OperationResultResponse<Guid> Execute(CreateUserRequest request)
 
             var dbUser = _mapperUser.Map(request, avatarImageId);
 
-            var password = request.Password ?? _generatePassword.Execute();
+            string password = !string.IsNullOrEmpty(request.Password) && request.Password.Trim().Any()
```

Comment:
Probably you can write it shorter like !string.IsNullOrEmpty(request.Password?.Trim()) and you have not to write &&

Reply:
And same in DbUserMapper.cs on 49 line

Reply:
Reworked